### PR TITLE
Update the fetch behaviour to work for multiple DEXs

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Update the fetch behaviour to work with the new pair ids configuration (introduced in #1095), which includes multiple DEXs.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [x] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: packages/valory/skills/apy_estimation_chained_abci/skill.yaml [hunk 1]
```diff
@@ -184,7 +183,7 @@ models:
       response_key: data
       response_type: dict
       retries: 5
-      url: https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2
+      url: https://api.thegraph.com/subgraphs/name/ianlapham/uniswapv2
```
Comment (dvilelaf) [inline packages/valory/skills/apy_estimation_chained_abci/skill.yaml:186]: Where does this url comes from?

File: packages/valory/services/apy_estimation_hardhat/service.yaml [hunk 2]
```diff
@@ -35,8 +35,7 @@ models:
         alpha: 0.25
       pair_ids:
         uniswap_subgraph:
-        - '0x5777d92f208679db4b9778590fa3cab3ac9e2168'
-        - '0xa374094527e1673a86de625aa59517c5de346d32'
```
Comment (dvilelaf) [inline packages/valory/services/apy_estimation_hardhat/service.yaml:39]: What does this change mean?

**Review by dvilelaf — COMMENTED** [review]
Waiting for tests, but looks good.

File: packages/valory/services/apy_estimation_hardhat/service.yaml (hunk 2 above)
Comment (Adamantios) [reply packages/valory/services/apy_estimation_hardhat/service.yaml:39]: I had accidentally used some v3 pool ids, so I replaced them with a v2.

File: packages/valory/skills/apy_estimation_chained_abci/skill.yaml (hunk 1 above)
Comment (Adamantios) [reply packages/valory/skills/apy_estimation_chained_abci/skill.yaml:186]: Copied it from the [official page](https://docs.uniswap.org/sdk/subgraph/subgraph-data).

**Comment by Adamantios** [comment]
Let's wait for #1134 to get approved and then see if merging it here solves the APY e2e issues. The e2e tests fail because of the optimization timing out, which #1134 attempts to solve, though this PR results in a more intense optimization round, because of the extra pool id added.

**Review by dvilelaf — DISMISSED** [review]

**Comment by Adamantios** [comment]
Opened #1168 for the failing e2e test.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**